core/src/banking_stage.rs

```diff
@@ -834,11 +830,6 @@ impl BankingStage {
                     starting_transaction_index = starting_index;
                 }
                 Err(PohRecorderError::MaxHeightReached) => {
-                    inc_new_counter_info!("banking_stage-max_height_reached", 1);
-                    inc_new_counter_info!(
-                        "banking_stage-max_height_reached_num_to_commit",
-                        num_to_record
-                    );
```

Comment:
seems this is actually removed? That's because it turned out that this counter is identical to `"banking_stage-record_transactions_retryable_record_txs"`?